frontend/src/components/CampoDePessoasComBuscaPorOrgao.vue

```diff
@@ -77,6 +77,7 @@ const props = defineProps({
   },
 });
 
+const { meta } = useRoute();
```

Comment:
Eu acho que essa linha dará problemas se a rota mudar, mas o componente for o mesmo.

Não deve acontecer tão logo no nosso código, mas prevenir é sempre bom.

Reply:
Não vejo porque poderia ser um problema.
Carregamos muita lógica no meta. O que muda é em vez de carregar rota.meta, estou desestruturando o meta direto.